src/strategies/index.ts

```diff
@@ -0,0 +1,10 @@
+import {FluentStrategy} from './FluentStrategy'
+import {Random, Dedupable, Biased, Cacheable, Shrinkable} from './FluentStrategyMixins'
+
+export const RandomCachedStrategy = Cacheable(Dedupable(Random(FluentStrategy)))
+
+export const RandomCachedStrategyWithShrinking = Shrinkable(Cacheable(Dedupable(Random(FluentStrategy))))
```

Comment:
Here, you prove what I said before. It's a cached strategy, not a cacheable strategy...

Reply:
Ok I will rename Chacheable to Cached